index.js

```diff
@@ -152,19 +152,19 @@ const main = async argv => {
   addConsole(args.logLevel);
   logger.debug(`Parsed args: ${JSON.stringify(args)}`);
 
-  const schemaKey = args.schema ? schemas[args.schema].$id : defaultSchema.$id;
+  const selectedSchema = schemas[args.schema];
```

Comment:
can not pass the other schema version ?

Reply:
This will be passed in via the schema flag

ie: `opencerts batch unsigned signed --schema 1.5`

If the flag is not set, it will default to the latest schema, currently `2.0`

Reply:
ok